desktop/index.js

```diff
@@ -19,6 +19,7 @@ app.use(require('./apps/collect_art'))
 app.use(require('./apps/collect'))
 app.use(require('./apps/categories'))
 app.use(require('./apps/categories2'))
+app.use(require('./apps/categories3'))
```

Comment:
Third time's a charm 😉 

I'll clean up `categories2`, which was the original attempt to productionize the 2016 hackathon project, after this gets merged.